dask_ml/wrappers.py

```diff
@@ -204,7 +204,9 @@ def transform(self, X):
         X = self._check_array(X)
 
         if isinstance(X, da.Array):
-            return X.map_blocks(_transform, estimator=self._postfit_estimator)
+            xx = np.zeros((1, X.shape[1]), dtype=X.dtype)
```

Comment:
Can you use a `np.zeros((0, X.shape[1]), dtype=X.dtype)` here? Or does scikit-learn not like that?

Reply:
Scikit-learn doesn't like it. 

```
ValueError: Found array with 0 sample(s) (shape=(0, 20)) while a minimum of 1 is required.
```